views/admin-quizzes-listing.phtml

```diff
@@ -90,8 +90,26 @@
                     </div>
                 </form>
             </div>
-            <div class="col-12 col-md-3 d-flex align-items-end justify-content-end">
-                <a class="btn btn-primary py-1" href="/admin-quiz-details/create">Add new quiz</a>
+            <div class="form-group col-12 col-md-2 d-flex align-items-end justify-content-end">
+                <form method="GET" action="/admin-quizzes-listing" >
+                    <?php foreach ($parameterBag->all() as $name => $value): ?>
+                        <?php if ($value !== '' && $name !== 'results'): ?>
+                            <?php  echo '<input type="hidden" name="' . $name . '" value="' . $value . '" />'; ?>
+                        <?php endif;?>
+                    <?php endforeach; ?>
+                    <label for="results">Results Per. Page</label>
+                    <div class="input-group">
+                        <input name="results" type="number" class="form-control" id="results" value="<?php echo $parameterBag->get('results')!==null ? $parameterBag->get('results') : 5;?>">
```

Comment:
Use simpler ternarnary: `echo $parameterBag->get('results') ?? 5`

Reply:
it was a mistake from copy pasting, the correct result is "echo $parameterBag->get('results')"